src/Cache.lua

```diff
@@ -463,7 +463,7 @@ local fieldConverters = {
 	["otherItemID"] = function(group, value)
 		CacheField(group, "itemID", value);
 	end,
-	["drakewatcherManuscriptID"] = function(group, value)
+	["mountModID"] = function(group, value)
```

Comment:
Can we make `mountModID` to `mountmodID` for the internal caching? This will make it inherently usable to chat prompt searches. Note this has to be changed everywhere it's used.